controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -109,3 +365,49 @@ func (r *StorageClassClaimReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&ocsv1alpha1.StorageClassClaim{}).
```

Comment:
You are missing an `Own` watch (similar to the `For` watch` for the storageclass type. 
Without a watch, this reconciler will not get triggered when a storageclass resource gets modified by another actor. 

Please add the needed `Own` watch